jb/itch5/order_book.hpp

```diff
@@ -42,7 +53,15 @@ typedef std::pair<price4_t, int> half_quote;
  * the total quantity available at that price?
  * - What is the best offer (lowest price of SELL orders) and what is
  * the total quantity available at that price?
+ *
+ * This is a template class.
+ * @param book_type define data structure type of price book,
+ * both sides buy and sell. Allowed values are:
+ * - jb::itch5::map_price
+ * @todo jb::itch5::cache_price
```

Comment:
I think I have been using "@todo(#bug_number) brief description".  Given that you plan to solve this quickly, maybe just remove the @todo.